service/models.py

```diff
@@ -135,13 +135,12 @@ def find(cls, by):
         logger.info("Processing lookup for id %s and condition %s ...", by_id, by_condition)
```

Comment:
I think you might have to add the test for find in test_models.py

Reply:
Yes! That has been included in the latest commit.